rollup.config.js

```diff
@@ -25,6 +25,16 @@ const getBabelOptions = ({ useESModules }, targets) => ({
   ],
 })
 
+function targetTypings(entry, out) {
+  return {
+    writeBundle() {
+      return fs.lstat(`dist/${out}`).catch(() => {
```

Comment:
```fs.lstat(`dist/${out}`).catch``` means "if `dist/${out}` doesn't exist".

We'd avoid writing `components.d.ts`, because `components/index.d.ts` already exists.